villas/controller/components/simulators/kubernetes.py

```diff
@@ -173,6 +176,9 @@ def _delete_job(self):
         self.job = None
         self.properties['job_name'] = None
         self.properties['pod_names'] = []
+        # job isn't immediately deleted
+        # let the user see that something is happening
+        time.sleep(7)
```

Comment:
Sorry, but this is a bad.. Why should we artifically block execution here?
This also stops the controller from processing other actions and stuff..